### DISCUSSION TITLE
Need help to release an email which is in use in other account(s)

### DISCUSSION THREAD

**Discussion by shawn-li-wcar** [community/community#43185] [New to GitHub]
Hello Github Community:

Could you please help me release the email [redacted due to PII]? So that I can change my primary email of this account (shawn-li-wcar) to it. 

When I try to add it to my shawn-li-wcar account, there is this error 'Error adding [redacted due to PII]: email is already in use'. This email is in use in shawn-li_wcar which I don't have the right to release it myself. (and maybe the account shawn-wcar is using this email too, which I lost my 2FA and I can't login into anymore)

Thanks in advice!

**Comment by radar-blip** [comment]
For account related issues, see here: [Support page](https://support.github.com/contact/account)

**Comment by queenofcorgis** [comment]
Hi @shawn-li-wcar,

I've removed your private information from this post as posting identifiable information is not allowed, per our [Code of Conduct](https://github.com/community/community/blob/main/CODE_OF_CONDUCT.md).

**Reply by shawn-li-wcar** [reply]
thank you!
